### PR TITLE
Add cubeviz docs for exporting data

### LINKED ISSUE #465: Add documentation for exporting data from Cubeviz
The section in RTD for Cubeviz -> Using Cubeviz -> Using Cubeviz in a Jupyter Notebook should be expanded to explain how to extract data from Cubeviz and use it elsewhere, like matplotlib.

### PR DESCRIPTION
**Pull request by javerbukh**
Resolves #465

### CODE REVIEW

File: docs/cubeviz/notebook.rst [hunk 1]
```diff
@@ -12,3 +12,25 @@ the following code in a cell of the notebook::
...
+`uncert-viewer`, and `mask-viewer`.
+Using the appropriate data label, the data in it's native type can be returned from this dictionary like
+so::
+
+    cubeviz.app.get_data_from_viewer('spectrum-viewer')['Subset 1']
+    cubeviz.app.get_data_from_viewer('flux-viewer')['contents[FLUX]']
+
+Data can also be accessed directly from ``data_collection`` using the following code::
+
+    cubeviz.app.data_collection[0]
```
Comment (duytnguyendtn) [inline docs/cubeviz/notebook.rst:34]: Super small, pedantic suggestion. I would add language to describe the indexing number can be used to index through all the available data (i.e. it doesn't have to be zero)

**Review by duytnguyendtn — APPROVED** [review]
One small, non-blocking suggestion. Otherwise, looks good!

File: docs/cubeviz/notebook.rst (hunk 1 above)
Comment (javerbukh) [reply docs/cubeviz/notebook.rst:34]: Good call!

File: docs/cubeviz/notebook.rst [hunk 2]
```diff
@@ -12,3 +12,27 @@ the following code in a cell of the notebook::
...
+connects to the ``glue-jupyter`` application level. This is accessed via the ``.app``
+attribute of the `~jdaviz.configs.cubeviz.helper.CubeViz` helper class. For example::
+
+     cubeviz.app.get_data_from_viewer('spectrum-viewer')
+     cubeviz.app.get_data_from_viewer('flux-viewer')
+
+This code can be used to access data from the different viewers, which is returned as a dictionary.
+The viewer options in the `cubeviz` configuration are `spectrum-viewer`, `flux-viewer`,
+`uncert-viewer`, and `mask-viewer`.
+Using the appropriate data label, the data in it's native type can be returned from this dictionary like
```
Comment (rosteen) [inline docs/cubeviz/notebook.rst:26]:
```suggestion
Using the appropriate data label, the data in its native type can be returned from this dictionary like
```
Minor spelling fix.

**Review by rosteen — APPROVED** [review]
I went ahead and committed one minor spelling fix, other than that I think this looks good.

**Pull request merged**